mathesar_ui/src/components/select/Select.svelte

```diff
@@ -28,22 +28,23 @@
 
   $: setOptions(options);
 </script>
-
-<Dropdown bind:isOpen contentClass="select">
-  <svelte:fragment slot="trigger">
-    {value?.[labelKey]}
+<span id="totalRows">Total Rows
```

Comment:
As mentioned in a previous comment, we could use `aria-label` instead of `aria-labelledby`, so we can remove this `span`.